### ISSUE TITLE
Parent pom.xml changed, change not picked up

### ISSUE THREAD

**Issue by stromnet** [danielflower/multi-module-maven-release-plugin#14]
I've made changes to my "root" pom.xml, and to a number of child modules (both pom and src).
The child changes are picked up fine, but the root pom change is not:

```
$ git diff lib-parent-1.0.1..HEAD pom.xml
diff --git a/pom.xml b/pom.xml
index 253c3a6..0592524 100644
--- a/pom.xml
+++ b/pom.xml
@@ -33,7 +33,7 @@
                <grizzly2.version>2.3.19</grizzly2.version>
 
                <!-- Newer than Jerseys -->
-               <jackson.version>2.5.1</jackson.version>
+               <jackson.version>2.6.1</jackson.version>
 
@@ -54,6 +54,7 @@
                <module>test-utils</module>
                <module>...</module>
                <module>...</module>
+               <module>utils</module>
        </modules>
 
        <dependencyManagement>
```

utils is a new module which did not exist before.

During releaser:release:

```
[INFO] Will use version 1.0.1 for lib-parent as it has not been changed since that release.
[INFO] Will use version 1.0.2 for test-utils as it has not been changed since that release.
...
[INFO] Will use version 1.0.0 for utils as it has changed since the last release.
...
```

The root project only has a pom.xml, no sources.

**Comment by stromnet** [comment]
I added a forceRelease option in #15 which helped me get around this problem:

mvn releaser:release -DforceRelease=lib-parent

Then all the submodules was properly re-released (and more importantly, the unit-tests which depended on jackson 2.6.1 did not fail and I actually got a full release working!)

**Comment by danielflower** [comment]
I really hope I can get to the bottom of this, but for now this is in 1.4.1

Thanks!

**Comment by danielflower** [comment]
Although I couldn't reproduce in a test, I think this _might_ be fixed in 1.4.2

If you get a chance to try it and it has solved it, let me know. Thanks!

**Comment by danielflower** [comment]
Closing this. I had seen this before 1.4.2 but haven't experienced it since, so I believe it's fixed.
